example-publish.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import pika
+
+with open('example-payload.json', 'r') as f:
+    data = f.read()
+
+connection = pika.BlockingConnection(pika.ConnectionParameters('localhost'))
+channel = connection.channel()
+
+channel.basic_publish(exchange='',
+                      routing_key=os.environ["TRAVIS_WORKER_QUEUE_NAME"]
+                      body=data)
+print(" [x] Sent 'Enqueued the thing!'")
+
+connection.close()
```

Comment:
nitpick: I think this script should be named *without* the `.py` extension (since it is not importable), and should instead be located at `./script/publish-example-payload`, also changing the `open` to read from an absolute path, e.g.:

``` python
# ...
import os

top = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))

with open(os.path.join(top, 'example-payload.json'), 'r') as f:
    data = f.read()

# ...
```